script/spec-runner.js

```diff
@@ -225,6 +225,9 @@ async function installSpecModules (dir) {
     npm_config_nodedir: nodeDir,
     npm_config_msvs_version: '2019'
   });
+  if (fs.existsSync(path.resolve(dir, 'node_modules'))) {
+    await fs.remove(path.resolve(dir, 'node_modules'));
+  }
```

Comment:
big oof

Reply:
We can remove this once my `node-gyp` changes are absorbed upstream